app/src/main/kotlin/com/wire/android/ui/home/conversationslist/bottomsheet/ConversationBottomSheet.kt

```diff
@@ -1,239 +1,143 @@
 package com.wire.android.ui.home.conversationslist.bottomsheet
 
-import androidx.annotation.DrawableRes
-import androidx.compose.foundation.clickable
-import androidx.compose.foundation.layout.ColumnScope
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.Spacer
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.height
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.layout.width
 import androidx.compose.material.ExperimentalMaterialApi
-import androidx.compose.material.TabRowDefaults.Divider
-import androidx.compose.material3.Icon
+import androidx.compose.material.ModalBottomSheetState
 import androidx.compose.material3.LocalContentColor
 import androidx.compose.material3.MaterialTheme
-import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.CompositionLocalProvider
-import androidx.compose.ui.Alignment
-import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.unit.Dp
-import androidx.compose.ui.unit.dp
 import com.wire.android.R
 import com.wire.android.ui.common.UserProfileAvatar
+import com.wire.android.ui.common.bottomsheet.MenuBottomSheetItem
+import com.wire.android.ui.common.bottomsheet.MenuItemIcon
+import com.wire.android.ui.common.bottomsheet.MenuModalSheetLayout
 import com.wire.android.ui.home.conversations.common.GroupConversationAvatar
-import com.wire.android.ui.theme.wireDimensions
-import com.wire.android.ui.theme.wireTypography
-import io.github.esentsov.PackagePrivate
+
 
 @OptIn(ExperimentalMaterialApi::class)
 @Composable
-fun ColumnScope.ConversationSheet(
+fun ConversationSheet(
+    sheetState: ModalBottomSheetState,
     modalBottomSheetContentState: ModalSheetContent,
     muteConversation: (String) -> Unit,
     addConversationToFavourites: (String) -> Unit,
     moveConversationToFolder: (String) -> Unit,
     moveConversationToArchive: (String) -> Unit,
     clearConversationContent: (String) -> Unit,
     blockUser: (String) -> Unit,
-    leaveGroup: (String) -> Unit
+    leaveGroup: (String) -> Unit,
```

Comment:
all these lambdas don't need `String` param, we can just make it `() -> Unit` and use the ID when calling the Sheet, like: 
`ConversationSheet(leaveGroup = { leaveGroup(id) } ... )`